xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css

```diff
@@ -167,6 +167,7 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
   color: $theme.textPrimaryColor;
   margin: 0;
   padding: 0 0 0 2px;
+  margin-top: 2px;
```

Comment:
bootstrap-select attaches itself in a `btn-group` div as a `btn` so bootstrap applies it's own styles on top of it, including a margin. Here's what it looks like:

![Margins](https://cloud.githubusercontent.com/assets/1671753/26240617/e7e40b72-3c81-11e7-97c1-0f088d5a4601.png)

It's probably better if we encapsulated all inputs in `btn-group`s but that's also more work, and doing so will mean we have to change a lot of styles... Another option is to just remove the margin altogether from the encapsulating div in the livetable, but this might break some bootstrap expectations. What do you think?